Wikipedia/Code/WMFWelcomeLanguageTableViewCell.swift

```diff
@@ -1,5 +1,5 @@
-
-class WMFWelcomeLanguageTableViewCell: WMFCustomDeleteButtonTableViewCell {
+//TODO: should this be themable?
```

Comment:
If welcome is always in the light theme, it doesn't need to be for now.

Reply:
That's what I suspected.